.travis.yml

```diff
@@ -8,9 +8,20 @@ python:
     - "3.6"
     - "3.7"
 
+env:
+    - TEST_DATABASE_URLS="postgresql://localhost/test_database, mysql://localhost/test_database, sqlite:///test.db"
+
+services:
+    - postgresql
+    - mysql
+
 install:
     - pip install -U -r requirements.txt
 
+before_script:
+    - psql -c 'create database test_database;' -U postgres
+    - echo 'create database test_database;' | mysql
+
 script:
     - scripts/test
```

Comment:
I wonder if we could do something like 
```yaml
script:
    - DATABASE_URL=postgresql://localhost/test_database scripts/test
    - DATABASE_URL=mysql://localhost/test_database scripts/test
    - DATABASE_URL=sqlite:///test.db scripts/test
```
and then avoid looping through databases.

Reply:
Yes, I agree that this would be a cleaner way to do it.

I'm also concerned that (at first glance at least) there seems to be a lot of code duplication here (either associated with database looping or not).

Reply:
actually I took same approach as https://github.com/encode/databases/blob/master/.travis.yml#L12

Reply:
@kiddick Yes, I think actually the issue is not the Travis config file but really the duplicated looping. It seems the databases repo uses `pytest.mark.parametrize` to make that cleaner — see [here](https://github.com/encode/databases/blob/master/tests/test_databases.py#L101).